src/Dominiks_modification.py

```diff
@@ -0,0 +1 @@
+print('Dominik is great!')
```

Comment:
Is he?

Reply:
Where's the new features of the the code?

Reply:
How do I change this?